mod_config_menu.lua

```diff
@@ -1543,4 +1563,28 @@ if MCMLoaded then
 			Info = {"Changes the color of error messages like Unknown pills"}
 		}
 	)
+	-- Mod indicator Color
+	MCM.AddSetting(
+		"EID",
+		"Colors",
+		{
+			Type = ModConfigMenu.OptionType.NUMBER,
+			CurrentSetting = function()
+				return AnIndexOf(colorNameArray, EID.Config["ModIndicatorTextColor"])
+			end,
+			Minimum = 0,
+			Maximum = 1000,
```

Comment:
Maximum = #colorNameArray

That way you dont need to to the custom handling of the selected table index in line 1583 and 1584

Reply:
I have to say I don't understand why there is a custom handling in other MCM options ?! I just copy/paste for this new option